src/containers/setup/user/ViewersContainer.js

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'react';
+import Components from '../../../components';
+import { connect } from 'react-redux';
+import Validator from 'validator';
+
+import List, {
+  ListItem,
+  ListItemText,
+} from 'material-ui/List';
+
+import Dialog, {
+  DialogContent,
+  DialogTitle,
+} from 'material-ui/Dialog';
+
+import TextField from 'material-ui/TextField';
+import Button from 'material-ui/Button';
+
+import Actions from "../../../actions";
+
+const Panel = Components.Misc.Panel;
+
+class NewViewerForm extends Component {
```

Comment:
This form doesn't follow the styling of the rest of the app

Reply:
👍
You can use `components/aws/accounts/FormComponent.js` as a template if needed.
In this component, we use `Dialogs` and `react-validation` (Instead of `material-ui/TextField`).